.changeset/better-meals-cough.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
@alexvy86 feedback welcome, merging now though. I can easily edit it in a second PR.